TF_Tooling_Web/.prettierignore

```diff
@@ -0,0 +1,3 @@
+/dist*
+/release
+src/TouchFree
```

Comment:
It might be worth excluding some of the example files as they are formatted the way they are for greater readability